src/App.test.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import renderer from 'react-test-renderer';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom/extend-expect';
+import Calculator from './components/Calculator';
+import Home from './components/home';
+import NavBar from './components/navbar';
```

Comment:
- [ ] Kindly write the path of the `NavBar` file right like that.
```javascript
import NavBar from './components/navBar';
```

Reply:
Hey @ShahierNashaat I think the path is correctly imported I tried importing it this way `import NavBar from './components/navBar';` and got some errors.